migrations/sqlite/20240530125008_event_header.up.sql

```diff
@@ -0,0 +1,9 @@
+-- Add up migration script here
+CREATE TABLE IF NOT EXISTS "ceramic_one_event_header" (
```

Comment:
How does this table get populated for existing dbs?

Reply:
I'm going to tackle that part today 😅. I'm planning to follow the kubo approach and basically add a CLI flag to not prompt, otherwise ask/exit if they should be applied. It will require reading every event, parsing, and storing the headers so it can't (easily) be done via a sql statement. I'm open to other approaches... was thinking I'd create a table like below and then query/update at start.

```sql
create table ceramic_one_data_migrations (
    version text unique not null,
    started_at timestamp not null default current_timestamp,
    completed_at timestamp
  );
  ```